tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+</code></pre>
+
+        <h4 style="font-weight: normal;">Closing Tags</h4>
+
+        <p>
+            In the past, developers traditionally used start and end tags for HTML elements. If we wanted some text on the page, we might 
+            create the following paragraph element <code>&lt;p&gt;Lorem ipsum dolor sit amet consectetur.&lt;/p&gt;</code>. With the advent 
+            of HTML5, it's not strictly necessary to close certain elements which are considered 
+            <a href="https://developer.mozilla.org/en-US/docs/Glossary/Void_element" target="_blank">void</a>, i.e., elements which can't have 
+            child nodes.
+        </p>
+
+        <p>
+            In HTML, it's possible to write a line-break element as <code>&lt;br&gt;</code> or <code>&lt;br /&gt;</code>. The trailing slash
+            in the tag has no meaning, and browsers simply ignore it. However, in XML, XHTML, and SVG, self closing tags are required in void elements, 
+            so a trailing slash is necessary. When working with tagged templates in JointJS, you must use a trailing slash. This can be 
+            observed in the following example:
+        </p>
+
+        <pre><code>&lt;foreignObject @selector="foreignObject"&gt;
+    &lt;div @selector="content"
+        xmlns="http://www.w3.org/1999/xhtml"
+        style="font-family: sans-serif;"
+    &gt;
+        &lt;p&gt;Lorem ipsum dolor sit amet consectetur.&lt;/p&gt;
+        &lt;br&gt; // line-break element without trailing slash will throw an error ❌
+        &lt;p&gt;Lorem ipsum dolor sit amet consectetur.&lt;/p&gt;
+    &lt;/div&gt;
+&lt;/foreignObject&gt;
+
+...
+
+// Use a trailing slash ✅
+&lt;br/&gt;
+</code></pre>
+
+        <h4 style="font-weight: normal;">HTML Entities</h4>
+
+        <p>
+            HTML Entities are used in place of some characters that are reserved depending on the language. For example,
+            the greater than (>) or less than (<) sign cannot be used in HTML as it would cause a conflict with tags.
+            This is also one area to be careful with when using tagged templates. You must use the entity number rather than
+            the entity name. Entity numbers are the ones which use the # symbol. That means using <code>&amp;#169;</code> 
+            is preferable over <code>&amp;copy;</code>.
+        </p>
+
+        <pre><code>&lt;foreignObject @selector="foreignObject"&gt;
+    &lt;div @selector="content" xmlns="http://www.w3.org/1999/xhtml"&gt;
+        &amp;copy; // copy HTML entity name will throw an error ❌
+    &lt;/div&gt;
+&lt;/foreignObject&gt;
+
+...
+
+// Use an entity number ✅
+&amp;#169;
+</code></pre>
+        
+        <h4>Browser related quirks</h4>
+
+        <p>
+            At the beginning of this tutorial, we mentioned how <code>foreignObject</code> is fully supported in all browsers.
+            Unfortunately, that doesn't mean some inconsistencies exist, or that some unexpected behaviour might occur. 
```

Comment:
the double-negative is confusing, should be rewritten, e.g. "Unfortuately, that is not completely true - some inconsistencies do exist, and some unexpected behavior may occur."